code-reviews/recipes/Markdown.md

```diff
@@ -0,0 +1,175 @@
+
+### markdownlint extension
+
+The [`markdownlint extension`](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) examines the markdown documents, showing warnings for rule violations while editing.
+
+## Build validation
+
+To automate linting with `markdownlint` for PR validation in GitHub actions, use the following YAML.
+
+```yaml
+name: Markdownlint
+
+on:
+  push:
+    paths:
+      - "**/*.md"
+  pull_request:
+    paths:
+      - "**/*.md"
+
+jobs:
+  lint:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: Run Markdownlint
+      run: |
+        npm i -g markdownlint-cli
+        markdownlint "**/*.md"
+```
+
+## Code Review Checklist
+
+Apart from the items on the [Code Review Checklist](../readme.md) you should also look for these markdown specific code review items
+
+- [ ] Is the document easy to read and understand and does it follow [good writing guidelines](#writing-style-guidelines)?
+- [ ] Is there a single source of truth or is content repeated in more than one document?
+- [ ] Is the documentation up to date with the code?
+
+## Writing Style Guidelines
+
+The following are some examples of writing style guidelines.
+
+Agree in your team which guidelines you should apply to your project documentation.
+Save your guidelines together with your documentation so they are easy to refer back to.
+
+### Wording
+
+- Use inclusive language, and avoid jargon and uncommon words. The docs should be easy to understand.
+- Be clear and consise, stick to the goal of the document.
+
+### Document Organization
+
+- Organize documents by topic rather than type, this makes it easier to find the documentation
+- Each folder should have a top-level readme.md and any other documents within that folder should link directly or indirectly from that readme.md
+- Document names with more than one word should use underscores instead of spaces, for example `machine_learning_pipeline_desgin.md`. The same applies to images.
+
+### Headings
+
+- Start with a H1 and respect the order H1 > H2 > H3 etc.
+- Avoid putting numbers in headings. Numbers shift, and can create outdated titles
+- Avoid using symbols and special characters in headers, this causes problems with anchor links
+- Avoid links in headers
+
+### Links
+
+- Avoid duplication of content, instead link to the `single source of truth`
+- Link but don't summarize. Summarizing content on another page leads to the content living in two places.
+- Use meaningful anchor texts, e.g. instead of writing `Follow the instructions [here](../recipes/Markdown.md)` write `Follow the [Markdown guidelines](../recipes/Markdown.md)`
+
+### Lists
+
+- List items should start with capital letters
```

Comment:
it's not possible in some cases, such as a list of some npm packages, etc. maybe you consider to add "if it's possible ..." in the beginning.

Reply:
fixed